jupyter_sphinx/__init__.py

```diff
@@ -34,6 +39,11 @@
 def skip(self, node):
     raise docutils.nodes.SkipNode
 
+# Used for nodes that should be gone by rendering time (OutputMimeBundleNode)
+def halt(self, node):
+    raise ExtensionError((f"Rendering encountered a node type that should "
```

Comment:
We list python 3.5 as the minimal requirement, so introducing f-strings also requires bumping the python version in setup and in travis config.

I personally don't mind bumping python to 3.6, but @choldgraf you know better the situation with its support across the board.

CC also @SylvainCorlay in case he has objectsions.

Reply:
I'm fine removing the f-string because I agree that deprecating 3.5 is a different issue than this PR addresses. My general strategy is to support the latest-3 versions of Python, unless there is a *really* good reason not to